sllurp/llrp_proto.py

```diff
@@ -184,25 +192,28 @@
 ROSpecState_Type2Name = reverse_dict(ROSpecState_Name2Type)
 
 # 10.2.1.1.1 ROSpec Start trigger
-StartTrigger_Name2Type = {
+ROSpecStartTriggerType_Name2Type = {
     'Null':                 0,
-    'Immediate':                1,
+    'Immediate':            1,
     'Periodic':             2,
     'GPI':                  3
 }
 
-StartTrigger_Type2Name = reverse_dict(StartTrigger_Name2Type)
+ROSpecStartTriggerType_Type2Name = reverse_dict(ROSpecStartTriggerType_Name2Type)
 
 # 10.2.1.1.2 ROSpec Stop trigger
 StopTrigger_Name2Type = {
     'Null':                 0,
     'Duration':             1,
-    'GPI with timeout':         2,
-    'Tag observation':          3
+    'GPI with timeout':     2,
+    'Tag observation':      3
 }
 
 StopTrigger_Type2Name = reverse_dict(StopTrigger_Name2Type)
+ROSpecStopTriggerType_Type2Name = StopTrigger_Type2Name
+AISpecStopTriggerType_Type2Name = StopTrigger_Type2Name
```

Comment:
Here, it could be interesting to profit of your changes to separate ROSpecStopTrigger and AISpecStop trigger into 2 different separated enums.

They are mostly similars with just the "Tag observation" as difference, but I think that if someone try to do enumerate values for a GUI, validation or just want to know what value is possible without looking at the specification, that could bite.